src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 
 2.0.25
 ++++++
+* msi: output principal id on enabling system assigned identity
```

Comment:
Double check... this will be part of next release?  Shouldn't this then be in 2.0.26?

Reply:
You are right. The PR missed the current release, so let me fix it up